### PR TITLE
Added Contributor to list

### PR DESCRIPTION
**Pull request by stevenanthonyrevo**
Hacktoberfest Participant Added to List

### CODE REVIEW

**Comment by stevenanthonyrevo** [comment]
@iamdarshshah Ready for a review and merge! Much appreciated!

File: src/contributors/stevenanthonyrevo.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Steven Anthony", 
+    "desc": "A Designer and Developer with a focus on GoLang, Swift and Node",
+    "github": "https://github.com/stevenanthonyrevo",
+    "twitter": "", 
+    "linkedin": ""
```
Comment (iamdarshshah) [inline src/contributors/stevenanthonyrevo.json:6]:
```suggestion
    "github": "https://github.com/stevenanthonyrevo"
```
Please don't add an empty string as a value for the param. Remove it, if you don't have the link.

**Review by iamdarshshah — CHANGES_REQUESTED** [review]

File: src/contributors/stevenanthonyrevo.json (hunk 1 above)
Comment (stevenanthonyrevo) [reply src/contributors/stevenanthonyrevo.json]: Thanks, I updated it with links!

**Review by iamdarshshah — APPROVED** [review]
This now LGTM! Thanks @stevenanthonyrevo

**Pull request merged**